samples/package.json

```diff
@@ -14,7 +14,7 @@
     "test": "repo-tools test run --cmd npm -- run cover"
   },
   "dependencies": {
-    "@google-cloud/dns": "0.6.2",
+    "@google-cloud/dns": "0.7.0",
```

Comment:
Is it a bad idea to put this as * (latest)?

Reply:
DPEs used to pin to specific versions to ensure correctness over time. With the repository merge, that may or may not make sense in the same way anymore (their tests will fail on our repo now, so we can not release without updating the samples). However, I do not want to rock the boat on their system immediately.